### ISSUE TITLE
mu4e support / instructions

### ISSUE THREAD

**Issue by kamushadenes** [seanfarley/emacs-bitwarden#8]
Hello,

How can this package be used with mu4e?

**Comment by seanfarley** [comment]
Do you mean sending mail (I use `'smtpmail`) or retrieving mail (I use `mbsync`)?

**Comment by kamushadenes** [comment]
Sending, I managed to make it work without any specific config by creating an entry in the correct format.

For future reference, you must create an entry in Bitwarden with the following:

Name: same as `smtpmail-smtp-server` (in my case, `smtp.gmail.com`)
Username: same as `smtpmail-smtp-user` (basically, your email address)
Password: well, your password. In case of GMail with MFA, this is the [App Password](https://myaccount.google.com/apppasswords)

Besides that, `auth-sources` must be (or contain) `'(bitwarden)`, as in `(setq auth-sources '(bitwarden))`.

Also, you obviously need `(bitwarden-auth-source-enable)` before setting auth-sources. In my experience, letting `bitwarden-auth-source-enable` set it, as it normally do, doesn't work for some reason.

Perhaps adding that to the README would be a good idea, here is an [example with lastpass](https://github.com/storvik/emacs-lastpass#auth-source-configuration-for-gmail).

I also managed to automatically unlock it by doing the following:

```
(defun get-file-contents (filename)
  "Return the contents of FILENAME."
  (with-temp-buffer
    (insert-file-contents filename)
    (buffer-string))) 

(setenv "BW_SESSION" (get-file-contents "~/.sensitive/bw_session"))
```

Where `~/.sensitive/bw_session` is a file containing only the value of BW_SESSION. This file is managed by some other script, or may even be written manually (not recommended).
